docs/testing.md

```diff
@@ -47,6 +47,13 @@ configuration is primarily managed through various onit commands like `onit set`
 file, but it's nevertheless important to note that the application must have write access to one
 of the above paths.
 
+
+### Bash or Zsh Auto-Completion
+*Onit* supports shell auto-completion for its various commands, sub-commands and flags. To enable this, run the following from the shell:
+
+```bash
+> eval "$(onit completion bash)"
```

Comment:
This needs to be `eval $(onit completion bash)` for bash and `source <(onit completion zsh)` for zsh. This also needs to be added to the `onit completion x` documentation as examples. And we can probably separate the bash and zsh sections here.

We should also recommend adding the output to `.bashrc` and `.zshrc`

Reply:
Sure. I will take care of it.

Reply:
This comment is addressed but there are other things that I should add or update